.github/workflows/publish.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish
+
+on:
+  push:
+    tags: 
+      - "v*.*.*"
+
+jobs:
+  publish:
+    name: Publish the package on pypi
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the source code
+        uses: actions/checkout@v2
+
+      - name: Set the python version
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
```

Comment:
Is the idea that we publish using the older supported Python version? It might be worth a comment if so, for future people looking at this.

Reply:
Added in the next commit.